### PR TITLE
Welcome 2017 Draft

### PR DESCRIPTION
**Pull request by nvgrw**
# DONT MERGE

# DONT MERGE

# DONT MERGE

# DONT MERGE

# DONT MERGE

Feedback pls

### CODE REVIEW

File: markdown/2017-10-02-welcome.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+### Bar Nights
+
+Oh, bar nights&hellip; Arguably one of DoCSoc's most known type of event, bar nights
+happen just about every month during term time. They're a great opportunity to
+talk to your coursemates and make new friends! Upon arrival, a volunteer will
+greet you and hand you tokens which can be redeemed for a vast array of drinks
+(both alcoholic and non-alcoholic) behind the bar. Ran out of tokens? Depending
+on the event, just ask a committee member to give you a few more tokens. We
+don't bite!
```
Comment (plietar) [inline markdown/2017-10-02-welcome.md]: Maybe add something like "they will be wearing DoCSoc t-shirts with their name on the back"

File: markdown/2017-10-02-welcome.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+
+Throughout the year you will also get to know your departmental representatives
+(Dep Reps). They represent the student body (you) to the department.
+
+This year they are:
+
+- **Computing**: Martin Zlocha
+- **Joint Maths and Computing (JMC)**: Rachel Mekhtieva Lee
+- **Electronics and Information Engineering (EIE)**: Mwanakombo Hussein
+
```
Comment (plietar) [inline markdown/2017-10-02-welcome.md]: Add email addresses (either personal or the generic depreps address)

File: markdown/2017-10-02-welcome.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+Want to get participate? Want to help run IC Hack? Details will follow later this
+term!
+We encourage all freshers to participate in their first year. Hackathons are a
+blast to attend and an amazing experience regardless of whether you've never
+been to a hackathon before or attend a different one very week.
+
+---
+
+Now for all the juicy details on our events this week(ish):
```
Comment (nvgrw) [inline markdown/2017-10-02-welcome.md]: not just our events (CodeCon)

File: markdown/2017-10-02-welcome.md [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+DoCSoc SF is looking for contributors! In July we started a new initiative which
+aims to give you the tools and support needed to contribute to Open Source, we
+call this the DoCSoc Software Foundation (DoCSoc SF). This is a great
+opportunity to help build tools which will be used by the society and
+help others gain experience in programming. If you're planning to apply for
+internships soon (or are already), contributing to open source projects can look
+great on your CV.
+
+Some of the current projects include:
+
```
Comment (plietar) [inline markdown/2017-10-02-welcome.md]: "you can propose project ideas as well"

File: markdown/2017-10-02-welcome.md (hunk 2 above)
Comment (plietar) [reply markdown/2017-10-02-welcome.md]: Also did we tell the EIE deprep we're including him here ?

Comment (nvgrw) [reply markdown/2017-10-02-welcome.md]: Good idea 
And no, I didn't do that. Should probably do that

File: markdown/2017-10-02-welcome.md (hunk 1 above)
Comment (nvgrw) [reply markdown/2017-10-02-welcome.md]: Not necessarily though. Don't Ops people volunteer too sometimes or is it only committee that hand out tokens?

File: markdown/2017-10-02-welcome.md [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+---
+
+# What is DoCSoc?
+DoCSoc is the Department of Computing Society at Imperial College London!
+
+This year it is brought to you by:
+
+- **President**: Maurice Yap
+- **Vice President**: Thomas Bower
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: I go by Tom. :)

File: markdown/2017-10-02-welcome.md [hunk 6]
```diff
@@ -0,0 +1,380 @@
...
+- **Electronics and Information Engineering (EIE)**: Mwanakombo Hussein
+
+For details on how to get involved, just get in touch with us. We're always
+looking for volunteers to help us set up and tear down the events. If you would
+like to volunteer, please join our [DoCSoc: Operations Facebook
+group](https://www.facebook.com/groups/1509850436007222/).
+
+## What Do We Do?
+DoCSoc is run by students, for students. With the generous support of our
+sponsors, we run events such as theatre trips, talks, and our infamous bar
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: infamous?

File: markdown/2017-10-02-welcome.md [hunk 7]
```diff
@@ -0,0 +1,380 @@
...
+nights (psst, all drinks are on DoCSoc)! Our membership base is mostly comprised
+of Computing, JMC, and EIE students, but membership can also be purchased on the
+[union
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: "Union" is capitalised.

File: markdown/2017-10-02-welcome.md [hunk 8]
```diff
@@ -0,0 +1,380 @@
...
+website](https://www.imperialcollegeunion.org/activities/a-to-z/computing) for
+anyone not on a DoC course. Why not get your friends on other courses to become
+members too?
+
+To stay up-to-date on the events that DoCSoc runs, keep an eye out for the
+weekly digest (you're reading it right now) which is baked and then delivered
+fresh to your inbox every week! Additionally, liking our [Facebook
+page](https://www.facebook.com/ICDocSoc/) will help you stay up to date on
+everything going on with DoCSoc.
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: Could you mention following on Twitter as well?

File: markdown/2017-10-02-welcome.md [hunk 9]
```diff
@@ -0,0 +1,380 @@
...
+
+## Our Sponsors
+Without the support of our generous sponsors, much of what we do and what makes
+DoCSoc so special would not be possible! Sponsors host many of our talks and
+will occasionally let you know of their great job opportunities.
+
+Which brings me on to why DoCSoc gets sponsorship: It's important to realise
+that our sponsors are looking to hire YOU! You get to build your career while we
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: Maybe embolden "YOU".

File: markdown/2017-10-02-welcome.md [hunk 10]
```diff
@@ -0,0 +1,380 @@
...
+get the money to run awesome social events. To kick things off, we will be
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: I probably wouldn't say "we get the money" - maybe something like "while we are given the tools to" or something. Just seems a little bit odd to start talking about the money straight away.

File: markdown/2017-10-02-welcome.md [hunk 11]
```diff
@@ -0,0 +1,380 @@
...
+department's ACI talks with our talks. At the end of a talk we usually hand out
+pizza for everyone! Veggie options are of course available.
+
+### IC Hack
+We run a hackathon every year called IC Hack.
+
+Check out the epicness of IC Hack 2017 on
+[YouTube](https://www.youtube.com/watch?v=VQl8tLcxptY)!
+
+Want to get participate? Want to help run IC Hack? Details will follow later this
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: Probably should be "Want to participate?" - the current doesn't make sense.

File: markdown/2017-10-02-welcome.md [hunk 12]
```diff
@@ -0,0 +1,380 @@
...
+term!
+We encourage all freshers to participate in their first year. Hackathons are a
+blast to attend and an amazing experience regardless of whether you've never
+been to a hackathon before or attend a different one very week.
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: very -> every

File: markdown/2017-10-02-welcome.md [hunk 13]
```diff
@@ -0,0 +1,380 @@
...
+- **Link:** [Facebook](https://www.facebook.com/events/1852626871714356/)
+
+Looking for an internship? Or want to talk directly to industry and our fabulous
+sponsors? Come join us in the Great Hall to have a chat with companies ranging
+from Facebook to Palantir and Netcraft to Microsoft in our premier autumn event.
+Bring lots of questions and be prepared to go home with a lot of swag!
+
+# Bloomberg CodeCon
+
+- **When:** 18.00-20.00, 13th October 2017
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: Shift this by half an hour. 18:30-20:30

File: markdown/2017-10-02-welcome.md [hunk 14]
```diff
@@ -0,0 +1,380 @@
...
+- **Where:** Skempton 060
+- **Link:** [Facebook](https://www.facebook.com/events/1816220712002800/)
+
+CodeCon is an annual coding competition put on by Bloomberg and run all around
+the world, including at Imperial. Come for fun or competition, and if you do
+well enough you cold make it to the global finals! Free food and drink will be
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: you cold -> you could

File: markdown/2017-10-02-welcome.md [hunk 15]
```diff
@@ -0,0 +1,380 @@
...
+
+Projects involve working on a range of commercial services extending Netcraft's
+network exploration and internet security services.
+
+To find out more, take a look at our recruitment booklet:
+<https://www.netcraft.com/about-netcraft/jobs/NetcraftGraduateBooklet.pdf>.
+
+Interested? Send a CV to <cv@netcraft.com>.
+
+# Bloomberg Blog Post
```
Comment (ThomasBower) [inline markdown/2017-10-02-welcome.md]: Are you only putting the logo for Netcraft? Why not the other companies too.

File: markdown/2017-10-02-welcome.md (hunk 3 above)
Comment (ThomasBower) [reply markdown/2017-10-02-welcome.md]: They are still "our" events, in that we host the sponsor. Tomato/tomato.

File: markdown/2017-10-02-welcome.md (hunk 11 above)
Comment (nvgrw) [reply markdown/2017-10-02-welcome.md]: oops i must've missed that while rewording cheers

Comment (nvgrw) [reply markdown/2017-10-02-welcome.md]: think it was 'want to get involved' initially

File: markdown/2017-10-02-welcome.md (hunk 13 above)
Comment (nvgrw) [reply markdown/2017-10-02-welcome.md]: I was using what was on the FB event which btw needs to be published once the digest is released https://www.facebook.com/events/1816220712002800/

File: markdown/2017-10-02-welcome.md (hunk 15 above)
Comment (nvgrw) [reply markdown/2017-10-02-welcome.md]: Netcraft and SIG asked for them

**Pull request merged**